spec/eof.md

```diff
@@ -279,6 +275,13 @@ Code executing within an EOF environment will behave differently than legacy cod
     - pop `offset` from the stack
     - if `offset + 32 > len(returndata buffer)`, execution results in an exceptional halt
     - push 1 item onto the stack, the 32-byte word read from the returndata buffer starting at `offset`
+- `CALL2 (0xf8)`, `DELEGATECALL2 (0xf9)`, `STATICCALL2 (0xfb)`
+    - Replacement of `CALL`, `DELEGATECALL` and `STATICCALL` instructions, as specced out in [EIP-7069](https://eips.ethereum.org/EIPS/eip-7069), except the runtime stack check. In particular:
+    - The `gas_limit` input is removed.
+    - The `output_offset` and `output_size` is removed.
+    - The `gas_limit` will be set to `(gas_left / 64) * 63` (aka as if the caller used `gas()` in place of `gas_limit`).
+
+    **NOTE**: Despite EOF not being strictly required by EIP-7069 and contrary to the wording of EIP-7069, the replacement instructions continue being treated as **undefined** in legacy code.
```

Comment:
I would get @axic's approval to merge this into the spec, he's the main author of EIP.

I would actually merge other changes separately without this.

Reply:
> I would actually merge other changes separately without this.

I think we can't. We must decide either "allowed in legacy" or "not allowed" and specify that explicitly. Current wording has it kinda blurry

Reply:
On the call we decided to not roll `*CALL2`s in legacy. I've only removed the references to EIP-7069 in the last sentence, they seemed confusing on 2nd thought.

Will merge this soon